core/citrus-api/src/main/java/org/citrusframework/spi/ResourcePathTypeResolver.java

```diff
@@ -116,10 +118,10 @@ public class ResourcePathTypeResolver implements TypeResolver {
     static {
         if (ROOT_IS_JAR) {
             try {
-                rootFs = FileSystems.newFileSystem(
-                    Path.of(ROOT.toString().substring("file:".length())));
+                rootFs = newFileSystem(
+                    new File(ROOT.toString().substring("file:".length())).toPath());
```

Comment:
Just curious. Why this? While is seems to serve the same purpose as the previous line, it mixes nio with io which seems less clean to me.